block/block.go

```diff
@@ -13,13 +14,31 @@ type Block struct {
 	Data []byte
 }
 
+// BlockRepository implementes data access layer for the Minetest map data.
 type BlockRepository interface {
+	// GetByPost returns the map block at positions X,Y,Z.
 	GetByPos(x, y, z int) (*Block, error)
+
+	// Iterator returns a channel to fetch all data from the starting position
+	// X,Y,Z, with the map blocks sorted by position ascending. Sorting is done
+	// by X,Y and Z coordinates.
+	Iterator(x, y, z int) (chan *Block, error)
```

Comment:
Could either receive a context to allow closing form it, or better yet, return a Closer interface value with a Close func. This way we could do early stop from the caller.

Reply:
> return a Closer interface

sounds good IMO :+1: